client/state/selectors/test/get-sites.js

```diff
@@ -95,7 +95,39 @@ describe( 'getSites()', () => {
 
 		expect( sites ).to.have.length( 3 );
 		expect( sites[ 0 ] ).to.have.property( 'ID', 2916288 );
-		expect( sites[ 1 ] ).to.have.property( 'ID', 2916287 );
-		expect( sites[ 2 ] ).to.have.property( 'ID', 2916289 );
+	} );
+
+	it( 'should returns sites in alphabetical order by name and url', () => {
```

Comment:
almost forgot about fixing this. Thanks! Changed on 7051b9bb0da91e6392c5a88035479a8698790cac.